app/stylesheet/application-webpack.scss

```diff
@@ -0,0 +1,6 @@
+$pf-global--enable-reset: false;
+$pf-global--disable-fontawesome: true;
+
+@import '~@fortawesome/fontawesome-free/css/all.css';
+@import '~@fortawesome/fontawesome-free/scss/v4-shims.scss';
+@import '~@patternfly/patternfly-next/patternfly.scss';
```

Comment:
should be `@patternfly/patternfly/patternfly.scss` - https://github.com/ManageIQ/manageiq-ui-classic/pull/5635

Reply:
Yes, it wasn't when I read the docs about it ...

Reply:
Not blaming you, I still can't even find any docs about it :)

But looks like the last release of patternfly-next was in January, since then, it's the other one.